setup.cfg

```diff
@@ -19,9 +19,8 @@ classifiers =
     Operating System :: POSIX :: BSD :: FreeBSD
     Operating System :: POSIX :: BSD :: NetBSD
     Operating System :: POSIX :: BSD :: OpenBSD
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
     Programming Language :: Python :: 3.4
```

Comment:
Need to remove the Python :: 3.4 and 3.5 as well.

Reply:
Good catch! Fixed in latest revision, commit "Fixup setup.cfg".